### PR TITLE
Added lowest deps tests

### PR DESCRIPTION
**Pull request by WouterJ**

### CODE REVIEW

**Comment by dbu** [comment]
do we have a problem with the mockery version?

**Comment by WouterJ** [comment]
Yes, and something with the publish workflow checker which I'm not sure why...

**Comment by WouterJ** [comment]
It's because `is_subclass_of($this->getMock('Symfony\Cmf\....\PublishTimePeriodReadInterface'), 'Symfony\Cmf\...\PublishTimePeriodReadInterface')` return false, while it should return true.

**Comment by dbu** [comment]
you could try building this with php 5.4 to see if its the php version or something else.

**Comment by WouterJ** [comment]
Hmm, it works with latest 5.3. Not sure if this means PHPunit mocks don't support 5.3.3 or if this bundle doesn't support it...

**Comment by dbu** [comment]
could it be an issue with the phpunit version we run? maybe the 5.3.3 image has an older version of that? can you try to fetch a phpunit.phar and run that instead of the provided, to see if that helps?

**Comment by WouterJ** [comment]
Both use PHPunit 4.5.0

File: composer.json [hunk 1]
```diff
@@ -18,9 +18,9 @@
     },
     "require-dev": {
         "symfony/security-bundle": "~2.1",
-        "mockery/mockery": "0.9.*",
-        "symfony-cmf/routing-bundle": "~1.2",
-        "symfony-cmf/testing": "~1.1",
+        "mockery/mockery": "~0.9.4",
+        "symfony-cmf/routing-bundle": "~1.2,>=1.2.0",
+        "symfony-cmf/testing": "dev-lowest_versions",
```
Comment (lsmith77) [inline composer.json]: should now require at least 1.2.6

**Comment by lsmith77** [comment]
green!

**Pull request merged**